.github/actions/generate-commit-list-v1/src/run.test.ts

```diff
@@ -49,17 +47,17 @@ describe('run', () => {
 
       assert.isTrue(setFailed.calledOnce)
       assert.isTrue(
-        setFailed.calledWith('Input required and not supplied: base')
+        setFailed.calledWith(
+          'You must provide either a tag or both a base and head branch.'
+        )
       )
     })
   })
 
   describe('when the head input is not provided', () => {
```

Comment:
Should have another test for when the tag input is not provided

Reply:
I think this test case covers the case when nothing is provided

https://github.com/dequelabs/axe-api-team-public/blob/feat-add-tags-commmit-list/.github/actions/generate-commit-list-v1/src/run.test.ts#L37-L55